src/routes/auth.js

```diff
@@ -18,4 +18,16 @@ router.get(
   authController.saveUserToTokenAndCookie
 );
 
+router.get('/facebook', passport.authorize('facebook', { scope: ['email'] }));
+router.get('/facebook', passport.authenticate('facebook'));
```

Comment:
I have a question here. Can these two lines be written like this:
`router.get('/facebook', passport.authorize('facebook', { scope: ['email'] }), passport.authenticate('facebook'));`
since these are two middlewares applied to the same router?

Reply:
I changed it like that and it worked. And also notice a comment from Muslim in the test file and delete it too.